docs/01-start-here/Quick visual glossary of the Guardian dot com.md

```diff
@@ -0,0 +1,108 @@
+# Quick visual glossary of the Guardian dot com 
```

Comment:
there is also some useful.duplicate info in great detail here: https://sites.google.com/a/guardian.co.uk/next-gen-fronts-design/home